### PR TITLE
Create delegate-standards.md

### PR DESCRIPTION
**Pull request by shelbeeee**

### CODE REVIEW

File: delegate-standards.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+All community members acting as delegates within the Radicle community are expected to adhere to the following set of standards in addition to the [Code of Conduct](https://docs.radworks.org/community/code-of-conduct). Any violation of these standards will be evaluated by Radicle community leaders using the Community Impact Guidelines outlined in the Conduct.
```
Comment (abbey-titcomb) [inline delegate-standards.md]:
```suggestion
All community members acting as delegates within the Radworks community are expected to adhere to the following set of standards in addition to the [Code of Conduct](https://docs.radworks.org/community/code-of-conduct). Any violation of these standards will be evaluated by Radworks community leaders using the Community Impact Guidelines outlined in the Conduct.
```

File: delegate-standards.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+In addition to the general community standards outlined in this code of conduct, community delegates are held to a set of more specific standards that support healthy governance.
+
+- Delegates should act at all times with honesty, integrity, and transparency.
+- Delegates should operate and vote in what they believe is in the best interest of the Radicle network & ecosystem.
```
Comment (abbey-titcomb) [inline delegate-standards.md]:
```suggestion
- Delegates should operate and vote in what they believe is in the best interest of the Radworks network & ecosystem.
```

File: delegate-standards.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+- Delegates should provide due care and attention in their role, making professional and unbiased reviews of each proposal prior to the submission of their vote.
+- Delegates should ensure that they communicate the rationale behind each of their votes in a clear and accessible way.
+  > 👉 Provide comments with thoughts and attitudes towards new proposals posted on the Discourse forum that make it to the “Formal Review” stage of the Radicle governance process. The status of the proposals is indicated in the title of the forum post.
```
Comment (abbey-titcomb) [inline delegate-standards.md]:
```suggestion
  > 👉 Provide comments with thoughts and attitudes towards new proposals posted on the Discourse forum that make it to the “Formal Review” stage of the Radworks governance process. The status of the proposals is indicated in the title of the forum post. 
```

File: delegate-standards.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+In addition to the general community standards outlined in this code of conduct, community delegates are held to a set of more specific standards that support healthy governance.
+
+- Delegates should act at all times with honesty, integrity, and transparency.
+- Delegates should operate and vote in what they believe is in the best interest of the Radicle network & ecosystem.
+- Delegates should provide due care and attention in their role, making professional and unbiased reviews of each proposal prior to the submission of their vote.
+- Delegates should ensure that they communicate the rationale behind each of their votes in a clear and accessible way.
+  > 👉 Provide comments with thoughts and attitudes towards new proposals posted on the Discourse forum that make it to the “Formal Review” stage of the Radicle governance process. The status of the proposals is indicated in the title of the forum post. 
+- Delegates should maintain a working knowledge of developments & happenings within in the Radicle ecosystem.
```
Comment (abbey-titcomb) [inline delegate-standards.md]:
```suggestion
- Delegates should maintain a working knowledge of developments & happenings within in the Radworks ecosystem.
```

**Review by abbey-titcomb — APPROVED** [review]
Just changed all "Radicle" to "Radworks"

**Review by pbjam12345 — APPROVED** [review]

**Pull request merged**